deck/packages/kubernetes/src/interfaces/infrastructure.types.ts

```diff
@@ -38,3 +38,10 @@ export interface IKubernetesServerGroupView extends IKubernetesServerGroup {
 }
 
 export interface IKubernetesServerGroupManager extends IServerGroupManager, IKubernetesResource {}
+
+export type IAnyKubernetesResource =
+  | IKubernetesServerGroup
```

Comment:
Hrmm, this seems a TOUCH strange to me but maybe I missed this... but won't there be types missing or is this JUST the "normal" vs. "Raw" types list?  Wasn't sure where "IKubernetesInstance" was used/defined in this PR?